src/storage_manager/index.js

```diff
@@ -35,6 +35,7 @@
 import defaults from './config/config';
 import LocalStorage from './model/LocalStorage';
 import RemoteStorage from './model/RemoteStorage';
+import callbackOrPromise from '../utils/callback-or-promise';
```

Comment:
We use this for the module resolution in webpack
```js
resolve: {
      modules: ['src', 'node_modules'],
	  ...
```
so you can do this ` from 'utils/callback-or-promise';
`